bucket/cairo-shell.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.4.80",
+    "description": "Cairo Desktop Environment",
+    "homepage": "https://cairoshell.com/",
+    "license": "Apache 2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/cairoshell/cairoshell/releases/download/v0.4.80/CairoSetup_64bit.exe#/dl.7z",
+            "hash": "b7a37dc28ec7a78f0906d92843619e431513719051e0186153779b3cf31ef2fd"
+        }
+    },
+    "bin": "CairoDesktop.exe",
```

Comment:
What are available command line parameters?

Reply:
https://github.com/cairoshell/cairoshell/wiki/Startup-Arguments

I am not sure if it is sensible to make them public via shortcut as they are probably rarely needed and only clutter up the start menu